app/controllers/ecosystems_controller.rb

```diff
@@ -128,4 +139,33 @@ def _create_response_payload_schema
     }
   end
 
+  def _fetch_metadatas_response_payload_schema
+    {
+      '$schema': JSON_SCHEMA,
+
+      'type': 'object',
+      'properties': {
+        'ecosystem_responses': {
+          'type': 'array',
+          'items': {'$ref': '#definitions/ecosystem_metadata'},
+          'minItems': 0,
+          'maxItems': 1000,
+        },
+      },
+      'required': ['ecosystem_responses'],
+      'additionalProperties': false,
+      'standard_definitions': _standard_definitions,
+      'definitions': {
+        'ecosystem_metadata': {
+          'type': 'object',
+          'properties': {
+            'uuid': {'$ref': '#standard_definitions/uuid'}
```

Comment:
Might wanna ask Kevin if he plans to filter ecosystems on UUID or if we need something like a book title/subject.